### PR TITLE
Complete homework_10

### PR DESCRIPTION
**Pull request by AnastasiiaNashchokina**

### CODE REVIEW

File: src/main/java/hw_10/Task1.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package hw_10;
+
+import jdk.internal.icu.text.UnicodeSet;
+
+import java.util.ArrayList;
```
Comment (AntBelyi) [inline src/main/java/hw_10/Task1.java:3]: Remove unused imports please

File: src/main/java/hw_10/Task1.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import java.util.Iterator;
+import java.util.List;
+
+public class Task1 {
+    public static void main(String[] args) {
+        List<String> list = new ArrayList();
```
Comment (AntBelyi) [inline src/main/java/hw_10/Task1.java]: add <> symbols: List<String> list = new ArrayList<>();

File: src/main/java/hw_10/Person.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+    @Override
+    public String toString() {
+        return "Person{" +
+                "name='" + name + '\'' +
+                ", surname='" + surname + '\'' +
+                ", age=" + age +
+                ", salary=" + salary +
+                '}';
+    }
```
Comment (AntBelyi) [inline src/main/java/hw_10/Person.java:51]: Please remove unused methods in Person class

File: src/main/java/hw_10/Task3.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        persons.add(new Person("Chris", "Black", 85, 25000));
+        for (Person person : persons){
+            System.out.println(person);
+        }
+        List<String> names = persons.stream()
+                .filter(person -> person.getAge() <70)
+                .map(Person::getName)
+                .toList();
+        System.out.println(names);
+    }
```
Comment (AntBelyi) [inline src/main/java/hw_10/Task3.java:25]: add extra lines between 16 - 17, 19 - 20, 23 - 24

File: src/main/java/hw_10/Person.java (hunk 3 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_10/Person.java:51]: fixed

File: src/main/java/hw_10/Task1.java (hunk 1 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_10/Task1.java:3]: fixed

File: src/main/java/hw_10/Task3.java (hunk 4 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_10/Task3.java:25]: fixed

File: src/main/java/hw_10/Person.java (hunk 3 above)
Comment (AntBelyi) [reply src/main/java/hw_10/Person.java:51]: not fixed

**Pull request merged**